### ISSUE TITLE
can this run in replit?

### ISSUE THREAD

**Issue by AstroOnYT** [DeadBread76/Raid-Toolbox#177]

**Comment by hidden-r4d** [comment]
No, one of the requirements takes too much memory for a repl to install.

**Comment by AstroOnYT** [comment]
well if i download this will the tool even work? and second of all does it have password/token grabbers in t

**Comment by AstroOrbis** [comment]
1. No grabbers
2. Join GitHub Education plan / dev plan to get free REPL Hacker plan. Disconnect and reconnect GitHub on your REPL account to redeem it after your plan gets activated.
